cmd/download.go

```diff
@@ -27,7 +32,8 @@ func Download(ctx *cli.Context) error {
 		os.Exit(1)
 	}
 
-	submission, err := client.Download(args[0])
+	submissionID := parseSubmissionID(args[0])
```

Comment:
We've got a bit of a mess in how we name things, but I think that we've been calling this the submission key rather than the submission ID.